app/src/main/java/com/cru_intern/geo_vicinity/base/db/RealmLiveData.kt

```diff
@@ -0,0 +1,35 @@
+package org.mpdx.base.realm
```

Comment:
This file wasn't being used because it had the wrong package name.
```suggestion
package com.cru_intern.geo_vicinity.base.db
```

Reply:
Fixed in new commit.